src/timeout.spec.ts

```diff
@@ -23,7 +23,7 @@ describe('Timeout', () => {
     });
 
     it('does not trigger callback again after completion', (done) => {
-        const timeout = new Timeout();
+        const timeout: Timeout = new Timeout();
```

Comment:
You shouldn't need `Timeout` declaration. I think the compiler should be able to infer the type.